.github/workflows/check.yml

```diff
@@ -76,6 +76,8 @@ jobs:
       matrix:
         python-version: [3.7, 3.8]
     runs-on: ubuntu-18.04
+    env:
+      SDK_E2E_TESTS_LOGS_DIR: build/e2e-home/stdout-logs/
```

Comment:
Can we put that in the makefile? We should capture these logs on local runs as well so people have easy access to them.